src/test/obj_persist_count/out2.log.match

```diff
@@ -1,7 +1,7 @@
 obj_persist_count$(nW)TEST2: START: obj_persist_count
  $(nW)obj_persist_count$(nW) $(nW)testfile
 task           cl(all) drain(all) pmem_persist pmem_msync pmem_flush pmem_drain pmem_memcpy_cls pmem_memcpy_drain pmem_memset_cls pmem_memset_drain potential_cache_misses 
-pool_create    62475   14         0            14         0          0          0               0                 0               0                 62475                  
+pool_create    67595   19         0            19         0          0          0               0                 0               0                 67595                  
```

Comment:
I think it depends on whether ndctl support is compiled or not. Try to build with NDCTL_ENABLE=n. If the test will pass without this patch, then you need to do something similar to what we did on x86 - see pool_create lines in out0.log.match.